README.md

```diff
@@ -10,8 +10,12 @@ SET SESSION DATABASE <your root database>;
 
```

Comment:
mam par bodu k organizaci kodu, ktere bych si rad ujasnil a pripadne sepsal sem do README pro dalsi vyvoj

- predpokladam ze vse v `Shared` je jen obecna omacka utils, co nemaji s konkretnimi drivery/backendy nic spolecneho. 
- Neni mi uplne jasne jak budou integrovatelne dalsi backendy. Todle je `php-storage-driver-teradata`, ale mame tady `Shared` a celou cast kolem protobufferu, kde u obou techto casti predpokladam, ze budou genericke/pouzitelne pro vsechny backendy implementovane touto formou
- No a pak je tady `src`, kterej je primo sita pro TD (organizace kodu apod). 

1. takze mame toto repo jako "docasne" dokud neudelame implementaci driveru v nejake jine technologii?
2. je to planovane jako TD only (src tak vypada) nebo sem budeme strkat dalsi backendy? Pokud TD only, tak by shared veci asi mely byt mimo

chtel bych tedy si ujasnit jaka je role tohoto repa v budoucnu (docasna/trvala, all backends/TD only) at si pro to zorganizujeme kod (slozky a namespacy) at pak nemame trable az rekneme "jdeme prepsat SNFLK/EXA... do driveru"

Reply:
Ano shared a generated a contract by měli mět vlastní repo, každopádně sem to hodil teď všecko do jednoho ať sa nezdržujeme omáčkama a řešením závislostí něčeho co je v takto ranné fázi. Jak sa to osvědčí a budeme dělat na dalším driveru tak sa to vycukne do vlastních repozitářů.

Reply:
Takže Shared,generated,Contract sů tady dočasně na návštěvě, protože je to Teradata only repo a žádný další driver sem nepůjde.